src/preload.py

```diff
@@ -649,7 +649,8 @@ def start_dockerd_and_wait_for_stdin(app_data, image=None):
         write_apps_json(app_data, mpoint + "/apps.json")
         with docker_context_manager(driver, mpoint):
             # Signal that Docker is ready.
-            print(json.dumps({}))
+            print(json.dumps({"statusCode": 0}))
+            sys.stdout.flush()
```

Comment:
This is uncomfortable, but it seems to work OK. Unfortunately, I found that there was additional communication between the CLI process and the Python process outside the `_runCommand` "framework", namely the `print(...)` statement above on the Python side, and the following `this.stdin.write('\n');` on the CLI side:

https://github.com/balena-io-modules/balena-preload/blob/v10.4.4/lib/preload.js#L1060-L1061